front/ui/ui-charts/src/speedSpaceChart/components/SpeedSpaceChart.tsx

```diff
@@ -174,52 +188,73 @@ const SpeedSpaceChart = ({
       )}
       <CurveLayer width={WIDTH_OFFSET} height={HEIGHT_OFFSET} store={store} />
       {store.layersDisplay.speedLimits && (
-        <SpeedLimitsLayer width={adjustedWidthRightAxis} height={height} store={store} />
+        <SpeedLimitsLayer width={adjustedWidthRightAxis} height={internalHeight} store={store} />
```

Comment:
Now, we can use pre-calculated values from `chartHeights` in the render function:


```suggestion
        <SpeedLimitsLayer width={adjustedWidthRightAxis} height={chartHeights.mainChartHeight} store={store} />
```

Reply:
Done